data/datasets/synthetic/anomalyPoint/phase/README.md

```diff
@@ -0,0 +1,15 @@
+# Corrupted data - point anomaly in phase
+
+An anomaly is added to an original clean data. In point anomalies we change the phaze in several (mostly four) consecutive samples for `pi`. This is aim to confuse the algorithm so that it quickly develops a false sense that the changed phase is the original anomaly free data. 
```

Comment:
Maybe reword it: 
`...In point phase anomalies we change the phase of the function at random points. We change it to a value dF (`0..pi`) for a few consecutive samples (mostly 4).  `

`Expectations: 
* `local memory`: the algorithm should detect the point of phase shift as an anomaly, but then quickly realize a known sequence and start predictions according to that. 
* `long-term memory`: alternatively, the algorithm should determine a known (sub)sequence (eg. downhill) and realize that it is not at the time when it should occur (there should be uphill).`